lib/cocoapods/installer/user_project_integrator.rb

```diff
@@ -113,6 +113,52 @@ def integrate_user_targets
         targets_to_integrate.sort_by(&:name).each do |target|
           TargetIntegrator.new(target).integrate!
         end
+
+        target_uuids_to_integrate = targets_to_integrate.flat_map(&:user_target_uuids).uniq
+        user_project_paths.each do |user_project_path|
+          project = Xcodeproj::Project.open(user_project_path)
+          
+          # TODO: Remove xcconfig files from targets_not_integrated
```

Comment:
Don't think it should in the current PR, should be addressed by CocoaPods/CocoaPods#2471